pep-8103.rst

```diff
@@ -1,11 +1,10 @@
 PEP: 8103
-Title: 2022 Term steering council election
-Version: $Revision$
-Last-Modified: $Date$
+Title: 2022 Term Steering Council election
 Author: Ewa Jodlowska <ewa@python.org>, Ee Durbin <ee@python.org>, Joe Carey <joe@python.org>
 Sponsor: Barry Warsaw <barry@python.org>
 Status: Active
```

Comment:
```suggestion
Status: Final
```

It seems I missed a spot—this term is over too (I'd gotten confused and thought this was the election _in_ 2022 instead of _for_ 2022, silly me).